src/Tools/BuildBoss/CompilerNuGetCheckerUtil.cs

```diff
@@ -107,7 +107,6 @@ private bool CheckPackages(TextWriter textWriter)
                     PathComparer.Equals(relativeFileName, "Icon.png") ||
                     PathComparer.Equals(relativeFileName, "Init.cmd") ||
                     PathComparer.Equals(relativeFileName, "VS.Tools.Roslyn.nuspec") ||
-                    PathComparer.Equals(relativeFileName, "csc.exe") ||
```

Comment:
@jaredpar should this actually have been vbc?

Reply:
Sure looks like it.

Reply:
I've updated the PR to exclude vbc.exe also instead of getting csc.exe excluded twice

Reply:
I'll wait for @jaredpar to take a look before merging, but otherwise LGTM

Reply:
It's definitely a typo: whether the intent was `csi` or `vbc` I cannot decide though.